lib/config/options/index.ts

```diff
@@ -2749,6 +2749,27 @@ const options: RenovateOptions[] = [
     cli: false,
     env: false,
   },
+  {
+    name: 'matchQueries',
+    description:
+      'JSON query to use. Valid only within a `customManagers` object of type `jsonata`.',
+    type: 'array',
+    subType: 'string',
+    parents: ['customManagers'],
+    cli: false,
+    env: false,
+  },
+  {
+    name: 'matchStrings',
+    description:
+      'Regex capture rule to use. Valid only within a `customManagers` object.',
+    type: 'array',
+    subType: 'string',
+    format: 'regex',
+    parents: ['customManagers'],
+    cli: false,
+    env: false,
+  },
```

Comment:
Was this missing before?

Reply:
No, I dupicated it accidentally and was confused all this while when the docs test was failing. Thanks

Reply:
We should have a test to catch such duplicates in the `options` object. 🤔